crates/ruff_python_parser/src/string.rs

```diff
@@ -616,6 +681,118 @@ mod tests {
         insta::assert_debug_snapshot!(suite);
     }
 
+    #[test]
```

Comment:
Same question as before regarding whether we need to add tests specific to t-string if there's any exclusive behavior that's not present for f-string.